frontend/src/segment.pug

```diff
@@ -0,0 +1,9 @@
+.segment(v-bind:class="{ untouched: !segment.authored, active: segment.lines.length<5 }")
+  .closer(v-if="!segment.authored && segment.lines.length>4", onclick="toggleNext(this)", v-on:click="loadCode()")
```

Comment:
This may be abstracted to `onclick="toggleNext(this); loaded = true"` so we do not need two listener for `on click` method. Or put `toggleNext(this)` inside `loadCode()`.